mods/ctf/ctf_bounties/init.lua

```diff
@@ -94,7 +84,13 @@ ctf.register_on_killedplayer(function(victim, killer)
 	end
 	bountied_player = nil
 
-	local msg = killer .. " has killed " .. victim .. " and received the prize!"
+	local killer_color = ctf_colors.get_color(ctf.player(killer)).css
+	local victim_color = ctf_colors.get_color(ctf.player(victim)).css
+	local msg = minetest.colorize(killer_color, killer) ..
+				minetest.colorize("#fff326"," has killed ") ..
+				minetest.colorize(victim_color, victim) ..
+				minetest.colorize("#fff326", " and received ") ..
+				minetest.colorize("#fff326", bounty_score .. " points!")
 	minetest.log("action", msg)
```

Comment:
`msg` contains colour-codes, which the would end up polluting the logs with stuff similar to `(C@fff326)`. The following would strip out the colour codes and log only the plain message.

```suggestion
	minetest.log("action", minetest.strip_colors(msg))
```